docs/zh_CN/CommunitySharings/feature_engineering.rst

```diff
@@ -0,0 +1,10 @@
+###################
+特征工程
+###################
+
+以下是一篇关于 NNI 如何助力特征工程的文章，由社区贡献者分享。 将来会添加更多用例和解决方案。
+
+..  toctree::
+    :maxdepth: 1
+
+    来自知乎的评论：作者 Garvin Li <NNI_AutoFeatureEng>
```

Comment:
where is the actual subject?

Reply:
docs/zh_CN/CommunitySharings/NNI_AutoFeatureEng.md